exasol_sagemaker_extension/lua/src/autopilot_endpoint_deployment.lua

```diff
@@ -70,6 +71,13 @@ function main(
 		instance_type, instance_count, aws_s3_connection, aws_region)
 	local script_schema_name = exa.meta.script_schema
 
+	if not validate_input.is_autopilot_endpoint_name_valid(endpoint_name) then
+		local error_obj = exaerror.create("E-SME-11",
+				"Invalid endpoint name " ..job_name
+		):add_mitigations("The name of endpoint should match the following pattern: ^(*[a-zA-Z0-9])")
```

Comment:
Since endpoint name is going to be connection name, I didn't want its name start with number. But, since we already using a prefix for that connection name, Its name can be start with number.